test/e2e-go/features/partitionRecovery/partitionRecovery_test.go

```diff
@@ -135,6 +135,10 @@ func runTestWithStaggeredStopStart(t *testing.T, fixture *fixtures.RestClientFix
 	// Stop Node1
 	nc1.FullStop()
 
+	status, err := fixture.LibGoalClient.Status()
+	a.NoError(err)
+	roundAfterStop := status.LastRound
```

Comment:
I haven't used this fixture, could you clarify what happens here? I see node 1 is running at this point and we wait until some rounds have been generated, then stop node 1.

goal is called to figure out which round we were at, but how does goal work when the node is stopped?

Reply:
It picks up the "Primary" node from the network, which is not stopped.